File: modul5/Python/PRAK505_2310817210017_MUHAMMAADNURHUDABAIHAQI.py
def biodata(tahun_lahir, namaku, asal):
    tahun_sekarang = 2023

    print(f"Perkenalkan Nama Saya : {namaku}")
    print(f"Umur Saya : {tahun_sekarang - tahun_lahir}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}\n")

if __name__ == "__main__":
    tahun_lahir = int(input())
    namaku = input()
    asal = input()

    biodata(tahun_lahir, namaku, asal)

File: modul5/C/PRAK505_2310817210017_MUHAMMAADNURHUDABAIHAQI.c
#include <stdio.h>
#include <string.h>

void Biodata(int tahunLahir, char Namaku[], char Asal[]) {
    int tahunSekarang = 2023;

    printf("Perkenalkan Nama Saya : %s\n", Namaku);
    printf("Umur Saya : %d\n", (tahunSekarang - tahunLahir));
    printf("Saya Adalah Angkatan : %d\n", tahunSekarang);
    printf("Asal Saya dari : %s\n\n", Asal);
}

int main() {
    int tahunLahir;
    char Namaku[30], Asal[15];

    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", Namaku);
    scanf(" %[^\n]%*c", Asal);

    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}
